debug/org.eclipse.debug.ui.launchview.tests/src/org/eclipse/debug/ui/launchview/tests/launchview/LaunchViewSmokeTest.java

```diff
@@ -23,10 +23,6 @@
 
 public class LaunchViewSmokeTest extends AbstractLaunchViewTest {
 
-	@SuppressWarnings({ "restriction", "unused" })
-	// Only ensures org.eclipse.debug.ui.launchview is required
-	private static final org.eclipse.debug.ui.launchview.IBackgroundLaunchExecutor e = null;
```

Comment:
Might better be replaced by `@Referenced` annotation, but the comment does not explain who needs this reference at all?